config/webpack.config.dev.js

```diff
@@ -1,4 +1,4 @@
-'use strict';
+/* eslint-disable */
 
```

Comment:
The webpack.configs are a bit big and confusing.

Reply:
right